### PR TITLE
Fix for RNN training on websockets, added instructions for Windows workaround installation

### PR DESCRIPTION
**Pull request by DanyEle**
## Description

- Fixed Recurrent Neural Network training on local and remote websockets in response to issue #4656 
- Currently, it is impossible to install PySyft in Windows because PyTorch 1.4.0 is not available. Added a workaround to install PySyft on Windows machines in INSTALLATION.md.

## Affected Dependencies
No additional dependencies affected

## How has this been tested?
- Working locally with websockets
- Test cases passing

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by gmuraru** [comment]
Thank you very much! :+1:

**Review by gmuraru — APPROVED** [review]

**Comment by DanyEle** [comment]
Ready to be merged?

**Comment by DanyEle** [comment]
Hello?

**Comment by gmuraru** [comment]
Yep - sorry. Do not know how I missed this :\
